### ISSUE TITLE
ArgumentError: namespace 'say' is already defined (OSX)

### ISSUE THREAD

**Issue by pdaugavietis** [michenriksen/gitrob#74]
Not sure where to start with this, but we'd like the running - i'll try on a Linux box tomorrow.

Error with bundle exec:

```
$ bundle exec gitrob configure
bundler: failed to load command: gitrob (/usr/local/lib/ruby/gems/2.3.0/bin/gitrob)
ArgumentError: namespace 'say' is already defined
  /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api/api.rb:301:in `namespace'
  /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api/client.rb:61:in `<class:Client>'
  /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api/client.rb:4:in `<module:Github>'
  /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api/client.rb:3:in `<top (required)>'
  /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api.rb:78:in `require'
  /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api.rb:78:in `block in require_all'
  /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api.rb:77:in `each'
  /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api.rb:77:in `require_all'
  /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api.rb:113:in `<module:Github>'
  /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api.rb:15:in `<top (required)>'
  /Users/pdaugavietis/data/source/loyaltyone/gitrob/lib/gitrob.rb:9:in `require'
  /Users/pdaugavietis/data/source/loyaltyone/gitrob/lib/gitrob.rb:9:in `<top (required)>'
  /Users/pdaugavietis/data/source/loyaltyone/gitrob/exe/gitrob:3:in `require'
  /Users/pdaugavietis/data/source/loyaltyone/gitrob/exe/gitrob:3:in `<top (required)>'
  /usr/local/lib/ruby/gems/2.3.0/bin/gitrob:23:in `load'
  /usr/local/lib/ruby/gems/2.3.0/bin/gitrob:23:in `<top (required)>'
```

With installed gem:

```
/usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api/api.rb:301:in `namespace': namespace 'say' is already defined (ArgumentError)
    from /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api/client.rb:61:in `<class:Client>'
    from /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api/client.rb:4:in `<module:Github>'
    from /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api/client.rb:3:in `<top (required)>'
    from /usr/local/Cellar/ruby/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:120:in `require'
    from /usr/local/Cellar/ruby/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:120:in `require'
    from /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api.rb:78:in `block in require_all'
    from /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api.rb:77:in `each'
    from /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api.rb:77:in `require_all'
    from /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api.rb:113:in `<module:Github>'
    from /usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api.rb:15:in `<top (required)>'
    from /usr/local/Cellar/ruby/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
    from /usr/local/Cellar/ruby/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
    from /usr/local/lib/ruby/gems/2.3.0/gems/gitrob-1.1.0/lib/gitrob.rb:9:in `<top (required)>'
    from /usr/local/Cellar/ruby/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
    from /usr/local/Cellar/ruby/2.3.0/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:68:in `require'
    from /usr/local/lib/ruby/gems/2.3.0/gems/gitrob-1.1.0/exe/gitrob:3:in `<top (required)>'
    from /usr/local/bin/gitrob:23:in `load'
    from /usr/local/bin/gitrob:23:in `<main>'
```

**Comment by sabsowitz** [comment]
I am having this exact same issue trying to install Gitrob on my Kali machine. Would really like to figure out what the problem is and how to fix it.

**Comment by pdaugavietis** [comment]
If i edit `/usr/local/lib/ruby/gems/2.3.0/gems/github_api-0.14.3/lib/github_api/client.rb` and comment out line 61 (namespace definition):
```
    namespace :repos
    alias :repositories :repos

    # namespace :say
    alias :octocat :say

    namespace :scopes
```
Seems to work lets me configure and scan.   I"ll submit a PR for it...

**Comment by sabsowitz** [comment]
@pdaugavietis I can confirm that your solution works. Thank you.

**Comment by johnconner31** [comment]
I am having same error and tried to edit the file as mentioned in referenced link, but didnt' found any namespace argument commenting on 61 line? Could you help me out if you managed it to work on your box?

Only difference i am seeing is you re using 

``github_api-0.14.3`` and i am ``github_api-0.14.4``

error i am getting is:

> /var/lib/gems/2.3.0/gems/github_api-0.14.4/lib/github_api/api.rb:301:in namespace': namespace 'say' is already defined (ArgumentError) from /var/lib/gems/2.3.0/gems/github_api-0.14.4/lib/github_api/client.rb:61:inclass:Client'
from /var/lib/gems/2.3.0/gems/github_api-0.14.4/lib/github_api/client.rb:4:in <module:Github>' from /var/lib/gems/2.3.0/gems/github_api-0.14.4/lib/github_api/client.rb:3:in'
from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in require' from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:inrequire'
from /var/lib/gems/2.3.0/gems/github_api-0.14.4/lib/github_api.rb:78:in block in require_all' from /var/lib/gems/2.3.0/gems/github_api-0.14.4/lib/github_api.rb:77:ineach'
from /var/lib/gems/2.3.0/gems/github_api-0.14.4/lib/github_api.rb:77:in require_all' from /var/lib/gems/2.3.0/gems/github_api-0.14.4/lib/github_api.rb:113:inmodule:Github'
from /var/lib/gems/2.3.0/gems/github_api-0.14.4/lib/github_api.rb:15:in <top (required)>' from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:inrequire'
from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in require' from /var/lib/gems/2.3.0/gems/gitrob-1.1.0/lib/gitrob.rb:9:in'
from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in require' from /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:inrequire'
from /var/lib/gems/2.3.0/gems/gitrob-1.1.0/exe/gitrob:3:in <top (required)>' from /usr/local/bin/gitrob:23:inload'
from /usr/local/bin/gitrob:23:in

**Comment by jasonstrange** [comment]
There have been a lot of these issues cropping up recently. Another possible solution is to `gem uninstall github_api` and then reinstall the specific version that's included in the gemfile. `gem install github_api -v 0.12` Duplicate w/ #78 and #77

**Comment by johnconner31** [comment]
@jasonstrange Thanks it worked.

**Comment by perfecto25** [comment]
have same issue on centos7, can confirm commenting out namespace say in client.rb fixes the issue.

**Comment by michenriksen** [comment]
This issue is fixed in version 1.1.1
